presto-docs/src/main/sphinx/release/release-0.181.rst

```diff
@@ -0,0 +1,68 @@
+=============
+Release 0.181
+=============
+
+General Changes
+---------------
+
+* Fix query failure and untracked memory usage when query contains
+  :func:`transform_keys` or :func:`transform_values`.
+* Fix a bug that may cause ``CREATE TABLE IF NOT EXISTS`` queries to fail
+  with *"Table already exists"*.
+* Fix query failure when ``ORDER BY`` expressions reference columns that are used in
+  the ``GROUP BY`` clause by their fully-qualified name.
+* Improve error handling when passing too many arguments to various
+  functions or operators that take a variable number of arguments.
+* Improve performance of ``count(*)`` aggregations over subqueries with known
+  constant cardinality.
+* Add support for ``VERBOSE`` modifier to ``EXPLAIN ANALYZE`` command for more detailed,
+  low-level information.
+* Add per-task distribution information to the output of ``EXPLAIN ANALYZE``.
+* Add support for ``DROP COLUMN`` in :doc:`/sql/alter-table`.
+* Change local scheduler to prevent starvation of long running queries
+  when the cluster is under constant load from short queries. The new
+  behavior is disabled by default and can be enabled on by setting the
+  config property ``task.level-absolute-priority=true``.
+* Improve fairness in the local scheduler such that long-running queries
+  which spend more time on the CPU per scheduling quanta (e.g., due to
+  slow connectors) do not get a disproportionate share of CPU. The new
+  behavior is disabled by default and can be enabled by setting the
+  config property ``task.legacy-scheduling-behavior=false``.
+* Add a config option to control prioritization of queries based on
+  elapsed scheduled time. The ``task.level-time-multiplier`` property
+  controls the target scheduled time of a level relative to the next
+  level. Higher values for this property increase the fraction of CPU
+  that will be allocated to shorter queries. This config property only
+  has an effect when ``task.level-absolute-priority=true`` and
+  ``task.legacy-scheduling-behavior=false``.
+
+
+Hive Changes
+------------
+
+* Fix potential native memory leak when writing tables using RCFile.
+* Correctly categorize certain errors when writing tables using RCFile.
+* Decrease file system metadata calls when reading tables.
+* Add support for dropping columns.
+
+
+JDBC Driver Changes
+-------------------
+
+* Support query cancellation using ``Statement.cancel()``.
+
+PostgreSQL Changes
+------------------
+
+* Add support for operations on external tables.
+
+Accumulo Changes
+----------------
+
+* Scan index ranges in parallel.
```

Comment:
This is a bit different, since we're not adding new functionality (some additional thing the user can do). This is an optimization. We could make this more clear by phrasing as
```
* Improve query performance by scanning index ranges in parallel.